packages/videomark-log-view/src/StatsSummary.jsx

```diff
@@ -36,7 +36,7 @@ const useStyles = makeStyles((theme) => ({
 }));
 const Stats = ({ title, type }) => {
   const classes = useStyles();
-  const baseUrl = new URL("https://sodium.webdino.org:8443/");
+  const baseUrl = new URL(process.env.BASE_URL+":8443/");
```

Comment:
文字列の中での変数の展開はテンプレートリテラル (https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals) を使うとより読みやすいと思います。

例:

```suggestion
  const baseUrl = new URL(`${process.env.BASE_URL}:8443/`);
```

Reply:
環境変数での展開を行う場合は、packages/videomark-log-view/config/webpack.config.js:L108

```js
// webpack.config.js
    new webpack.EnvironmentPlugin(["NODE_ENV"]),
```

↓

```js
// webpack.config.js
    new webpack.EnvironmentPlugin(["NODE_ENV", "BASE_URL"]),
```

のように、使用する変数を明示しておく必要があるのではないでしょうか?
参考文献: https://webpack.js.org/plugins/environment-plugin/

Reply:
ありがとうございます。
参考文献の閲覧も含めて、進めていきます。